Packages/vcs/vcs/utils.py

```diff
@@ -312,42 +312,6 @@ def _scriptrun(script, canvas=None):
                 vcs.process_src_element(s)
                 processing = False
     f.close()
-    # Ok now we need to double check the isolines
-    gd = vcs.elements["isoline"]["default"]
-    for g in vcs.elements["isoline"].values():
-        if g.name == "default":
-            continue
-        for att in ["line", "textcolors", "text"]:
-            try:
-                setattr(g, att, getattr(g, att))
-            except:
-                lst = []
-                if att == "line":
-                    for e in g.line:
-                        if e in vcs.elements["line"]:
-                            lst.append(vcs.elements["line"][e])
-                        else:
-                            lst.append(e)
-                elif att == "text":
-                    for e in g.line:
-                        if e in vcs.elements["textorientation"]:
-                            lst.append(vcs.elements["line"][e])
-                        elif e in vcs.elements["textcombined"]:
-                            lst.append(vcs.elements["line"][e])
-                        else:
-                            lst.append(e)
-                elif att == "textcolors":
-                    for e in g.line:
-                        if e in vcs.elements["texttable"]:
```

Comment:
i think you're right it should be `g.text` and `g.textcolor` looks like some copy/pasting that went bad.